OpenMEEG/CMakeLists.txt

```diff
@@ -185,7 +185,8 @@ add_subdirectory(Wrapping)
 #   Tests
 
 if (BUILD_TESTING)
-    include(CreateTestEnvironment)
+    # XXX : this breaks when trying to build OpenMEEG without the project :(
+    # include(CreateTestEnvironment)
```

Comment:
@papadop  this happens otherwise:

https://travis-ci.org/openmeeg/openmeeg/jobs/130379872